README.md

```diff
@@ -1,4 +1,16 @@
 # botamusique
+
+**Info:**
```

Comment:
What do you think about quoting this addition? This might make your note a bit more prominent.

By quoting I mean a `>` before a line:

> Quoting this paragraph is just a suggestion.

Keep up the good work 😄.